ui/less/general.less

```diff
@@ -37,7 +37,7 @@
 }
 
 .bottom-panels-elements-margin() {
-  margin: 1px 6px;
+  margin: 1px;
```

Comment:
We have to remain 1 px margin to render outline because outline is drawn outside the element's frame.